daemon/containerd/image_exporter.go

```diff
@@ -342,3 +378,62 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, platf
 
 	return nil
 }
+
+// verifyImagesProvidePlatform checks if the requested platform is loaded.
+// If the requested platform is not loaded, it returns an error.
+func (i *ImageService) verifyImagesProvidePlatform(ctx context.Context, imgs []containerdimages.Image, platform ocispec.Platform, pm platforms.Matcher) error {
```

Comment:
Wondering; should an empty `imgs` slice also return an error? I _THINK_ currently an empty `imgs` slice also will result in an empty `imageNames` slice ( "no images missing"), and because of that return "no error"

Reply:
Don't think so. Empty `imgs` should probably be handled by the caller of `client.Import`. On the other hand - that would disallow loading "empty" image indexes, which we do for some tests.

Reply:
Ah right, done